cmd/memberagent/main.go

```diff
@@ -160,8 +160,23 @@ func buildHubConfig(hubURL string, useCAAuth bool, tlsClientInsecure bool) (*res
 
 	hubConfig.TLSClientConfig.Insecure = tlsClientInsecure
 	if !tlsClientInsecure {
-		hubConfig.TLSClientConfig.CAFile = os.Getenv("CA_BUNDLE")
-		hubCA := os.Getenv("HUB_CERTIFICATE_AUTHORITY")
+		caBundle, ok := os.LookupEnv("CA_BUNDLE")
+		if ok && caBundle == "" {
+			err := errors.New("environment variable CA_BUNDLE should not be empty")
+			klog.ErrorS(err, "failed to validate system variables")
+		}
+		hubCA, ok := os.LookupEnv("HUB_CERTIFICATE_AUTHORITY")
+		if ok && hubCA == "" {
+			err := errors.New("environment variable HUB_CERTIFICATE_AUTHORITY should not be empty")
+			klog.ErrorS(err, "failed to validate system variables")
+		}
+		if caBundle != "" && hubCA != "" {
+			err := errors.New("environment variables CA_BUNDLE and HUB_CERTIFICATE_AUTHORITY should not be set at same time")
+			klog.ErrorS(err, "failed to validate system variables")
+			return nil, err
+		}
+
+		hubConfig.TLSClientConfig.CAFile = caBundle
```

Comment:
we should only set caBundle or caData

Reply:
fixed in [this commit](https://github.com/Azure/fleet/pull/365/commits/84e6d516bad972a4345acfc5894a9c2bab42998d)